Libraries/SPTarkov.Server.Core/Models/Spt/Mod/AbstractModMetadata.cs

```diff
@@ -52,6 +52,7 @@ public abstract record AbstractModMetadata
     /// <br/>
     /// Version = new Version("4.0.0.0"); is not
     /// </summary>
+    [Obsolete("Will be removed in 4.1, we now check the mods assembly reference to SPTarkov.Server.Core during mod loading")]
```

Comment:
I would advise against removing this. You are only doing a minVer check, the SptVersion property is used to do a full semVer range check, which is both a minimum and maximum